### ISSUE TITLE
Fix collector build error with nightly toolchain

### ISSUE THREAD

**Issue by Dicarbene** [rust-lang/rustc-perf#1821]
While building rustc-perf/collector using toolchain :

```rust
info: syncing channel updates for 'nightly-x86_64-unknown-linux-gnu'
info: latest update on 2024-02-22, rust version 1.78.0-nightly (3406ada96 2024-02-21)
```
I get error like this.

```rust
error[E0635]: unknown feature `stdsimd`
  --> /home/dicarbene/.cargo/registry/src/mirrors.ustc.edu.cn-61ef6e0cd06fb9b8/ahash-0.8.6/src/lib.rs:99:42
   |
99 | #![cfg_attr(feature = "stdsimd", feature(stdsimd))]
   |
```
I find that stdsimd is deleted in the latest version
![image](https://github.com/rust-lang/rustc-perf/assets/53905542/fa165ad7-25f0-4128-8e17-6d042a5639ec)

[error[E0635]: unknown feature stdsimd](https://users.rust-lang.org/t/error-e0635-unknown-feature-stdsimd/106445/6)

I'm working on it to see if we could  bump up some lib version to fix this.

If anyone encounter this error, try fix it building with stable toolchain or run  `cargo update ahash` should help

**Comment by Kobzol** [comment]
`cargo tree -i ahash` should be helpful to see what packages depend on the old ahash version. But in any case, ahash should now have minor releases with a fix for both 0.7 and 0.8, so just updating ahash should hopefully work.

**Comment by Dicarbene** [comment]
> `cargo tree -i ahash` should be helpful to see what packages depend on the old ahash version. But in any case, ahash should now have minor releases with a fix for both 0.7 and 0.8, so just updating ahash should hopefully work.

Thanks for the help, this is the dependencies tree.
```shell
ahash v0.8.9
├── hashbrown v0.14.3
│   ├── database v0.1.0 (/home/dicarbene/rustc-perf/database)
│   │   ├── collector v0.1.0 (/home/dicarbene/rustc-perf/collector)
│   │   │   └── site v0.1.0 (/home/dicarbene/rustc-perf/site)
│   │   └── site v0.1.0 (/home/dicarbene/rustc-perf/site)
│   ├── hashlink v0.8.4
│   │   └── rusqlite v0.28.0
│   │       └── database v0.1.0 (/home/dicarbene/rustc-perf/database) (*)
│   ├── indexmap v2.1.0
│   │   ├── h2 v0.3.22
│   │   │   ├── hyper v0.14.25
│   │   │   │   ├── hyper-tls v0.5.0
│   │   │   │   │   └── reqwest v0.11.14
│   │   │   │   │       ├── collector v0.1.0 (/home/dicarbene/rustc-perf/collector) (*)
│   │   │   │   │       ├── database v0.1.0 (/home/dicarbene/rustc-perf/database) (*)
│   │   │   │   │       └── site v0.1.0 (/home/dicarbene/rustc-perf/site)
│   │   │   │   ├── reqwest v0.11.14 (*)
│   │   │   │   └── site v0.1.0 (/home/dicarbene/rustc-perf/site)
│   │   │   └── reqwest v0.11.14 (*)
│   │   ├── rust_team_data v1.0.0 (https://github.com/rust-lang/team#89bb40d3)
│   │   │   └── site v0.1.0 (/home/dicarbene/rustc-perf/site)
│   │   └── toml_edit v0.19.15
│   │       └── toml v0.7.8
│   │           └── site v0.1.0 (/home/dicarbene/rustc-perf/site)
│   │           [build-dependencies]
│   │           └── site v0.1.0 (/home/dicarbene/rustc-perf/site)
│   ├── intern v0.1.0 (/home/dicarbene/rustc-perf/intern)
│   │   ├── collector v0.1.0 (/home/dicarbene/rustc-perf/collector) (*)
│   │   └── database v0.1.0 (/home/dicarbene/rustc-perf/database) (*)
│   ├── lru v0.12.0
│   │   └── site v0.1.0 (/home/dicarbene/rustc-perf/site)
│   └── site v0.1.0 (/home/dicarbene/rustc-perf/site)
└── inferno v0.11.15
    └── site v0.1.0 (/home/dicarbene/rustc-perf/site)
```
I guess right now the best solution is use the temperal fix `cargo update ahash`, and wait till upstream libs upgrade their dependencies, then we bump up the depencies version.
![image](https://github.com/rust-lang/rustc-perf/assets/53905542/60f9b774-3dff-4923-9e88-8421ebc498c0)
